components/_data/data.json

```diff
@@ -15,80 +15,73 @@
+  "excerpt": {
+    "short": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
+    "medium": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    "long": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
   },
-  "description" : "So, setting about it as methodically as men might smoke out a wasps' nest, the Martians spread this strange stifling vapour over the Londonward country. The horns of the crescent slowly moved apart, until at last they formed a line from Hanwell to Coombe and Malden. All night through their destructive tubes advanced.",
-  "url" : "#",
-  "name" : {
+  "description": "So, setting about it as methodically as men might smoke out a wasps' nest, the Martians spread this strange stifling vapour over the Londonward country. The horns of the crescent slowly moved apart, until at last they formed a line from Hanwell to Coombe and Malden. All night through their destructive tubes advanced.",
+  "url": "#",
+  "name": {
     "first": "Lacy",
     "firsti": "L",
     "middle": "Tommie",
     "middlei": "T",
     "last": "Way",
     "lasti": "W"
   },
-  "year" : {
+  "year": {
     "long": "2013",
     "short": "13"
   },
-  "month" : {
+  "month": {
     "long": "February",
     "short": "Feb",
     "digit": "02"
   },
-  "dayofweek" : {
+  "dayofweek": {
     "long": "Monday",
     "short": "Mon"
   },
-  "day" : {
+  "day": {
     "long": "10",
     "short": "10",
     "ordinal": "th"
   },
-  "hour" : {
+  "hour": {
     "long": "01",
     "short": "1",
     "military": "13",
     "ampm": "pm"
   },
-  "minute" : {
+  "minute": {
     "long": "20",
     "short": "20"
   },
-  "seconds" : "31",
-  "author" : {
+  "seconds": "31",
+  "author": {
     "first_name": "Author",
     "last_name": "Name"
   },
   "hero": true,
-  "emergency" : false,
-  "touts" : [
-    { },
-    { },
-    { }
-  ],
-  "latest_posts" : [
-    { },
-    { },
-    { },
-    { },
-    { }
-  ]
+  "emergency": false,
+  "touts": [{}, {}, {}],
```

Comment:
What is this?

Reply:
This is a global data file that's only used by Pattern Lab. It doesn't have any affect on the site, but I added a global "alt" text default, just so that Pattern Lab wouldn't throw false-negatives in the accessibility check.

If that's not clear, we can set up a chat to talk it through

Reply:
I meant specifically `touts`—it simply wasn't a word I recognized in English.